Sources/Subprocess/Execution.swift

```diff
@@ -35,16 +35,13 @@ public struct Execution: Sendable {
     public let processIdentifier: ProcessIdentifier
 
     #if os(Windows)
-    internal nonisolated(unsafe) let processInformation: PROCESS_INFORMATION
     internal let consoleBehavior: PlatformOptions.ConsoleBehavior
```

Comment:
nit: unrelated, but you could delete `consoleBehavior` as well as nothing actually uses it.

Reply:
I originally left it there because I wasn't sure if it is useful... I agree with you that it seems not that useful. I'll remove it in a dedicate PR since it's an API change.